charts/xapp-anr/values.yaml

```diff
@@ -0,0 +1,68 @@
+global: {}
+
+nameOverride: ""
+fullnameOverride: ""
+
+replicaCount: 1
+
+bootstrap:
+  create: true
+  name: ""
+  kafka:
+    enabled: true
+    hostname: ""
```

Comment:
It's maybe worth looking into #142 first so we would get an error message when this field is not provided.

Reply:
OK - do I have to change anything in this part of the values file?

Reply:
No, it's already good